tests/src/FunctionalJavascript/WebformCivicrmTestBase.php

```diff
@@ -143,6 +148,40 @@ protected function assertPageNoErrorMessages() {
     }, $error_messages)));
   }
 
+  /**
+   * Copy of TraversableElement::fillField, but it replaces the existing value on the element rather than appending to it.
+   *
+   * Fills in field (input, textarea, select) with specified locator.
+   *
+   * @param string $locator input id, name or label
+   * @param string $value   value
+   *
+   * @throws ElementNotFoundException
+   *
+   * @see NodeElement::setValue
+   */
+  public function addFieldValue($locator, $value) {
+    $field = $this->getSession()->getPage()->findField($locator);
+    if (null === $field) {
+      throw new ElementNotFoundException($this->getDriver(), 'form field', 'id|name|label|value|placeholder', $locator);
+    }
+    $field->doubleClick();
+    $field->setValue($value);
+  }
+
+  /**
+   * Assert populated values on the field.
+   * fieldValueEquals() fails for populated values on chromedriver > 91
+   *
```

Comment:
Just noting that it still works fine for load and check (assert). Things only go sideways when we're testing to update/rewrite a field.